config.js

```diff
@@ -637,6 +637,9 @@ config.INLINE_MAX_SIZE = 4096;
 // Object SDK bucket cache expiration time
 config.OBJECT_SDK_BUCKET_CACHE_EXPIRY_MS = 60000;
 
+// Object SDK bucket_namespace_cache allow stat of the config file
+config.ALLOW_BUCKET_NS_CACHE_STAT = true;
```

Comment:
I think we should mention here NC, because the stat is on config files and that it's for validation
```suggestion
config.NC_ENABLE_BUCKET_NS_CACHE_STAT_VALIDATION = true;
```